src/main/java/com/google/sps/servlets/ImageServlet.java

```diff
@@ -52,26 +50,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     PrintWriter writer = response.getWriter();
     BlobKey blobKey = getBlobKey(request, "image");
-    if (blobKey == null) {
+    if (blobKey == null) {  // Not an image file.
```

Comment:
This will be null if nothing was uploaded, but still a non image file could be passed.

Reply:
I will add a TODO for now.